### PR TITLE
Fix duplicate objects

### PR DESCRIPTION
**Pull request by viclovsky**
Fix issue of generating duplicated schema classes in case when the same schema path is specified in raml file. 
After fix all schema classes will be generated in one exact folder and will be imported for dependent files.

### CODE REVIEW

**Comment by lanwen** [comment]
pls few words to describe content

File: rarc-core/src/main/java/ru/lanwen/raml/rarc/api/ApiResourceClass.java [hunk 1]
```diff
@@ -15,15 +16,8 @@
...
-import static java.util.stream.Collectors.groupingBy;
-import static java.util.stream.Collectors.joining;
-import static java.util.stream.Collectors.toList;
-import static org.apache.commons.lang3.StringUtils.capitalize;
-import static org.apache.commons.lang3.StringUtils.isNotEmpty;
-import static org.apache.commons.lang3.StringUtils.substringAfterLast;
-import static org.apache.commons.lang3.StringUtils.trimToEmpty;
-import static org.apache.commons.lang3.StringUtils.uncapitalize;
-import static org.apache.commons.lang3.StringUtils.upperCase;
+import static java.util.stream.Collectors.*;
```
Comment (lanwen) [inline rarc-core/src/main/java/ru/lanwen/raml/rarc/api/ApiResourceClass.java]: please dont

Comment (viclovsky) [reply rarc-core/src/main/java/ru/lanwen/raml/rarc/api/ApiResourceClass.java]: ok

**Comment by viclovsky** [comment]
Fix issue of generating duplicated schema classes in case when the same schema path is specified in raml file. 

After fix all schema classes will be generated in one exact  folder and will be imported for dependent files.

**Comment by lanwen** [comment]
please don't use wildcard imports. And I'll merge it

**Review by lanwen — CHANGES_REQUESTED** [review]
rm wildcard imports

File: rarc-example/src/main/resources/api.raml [hunk 2]
```diff
@@ -107,6 +107,19 @@ traits: !include /traits.yaml
               enumparam_in_form:
                 description: enum parameter in form parameters should not became query parameter and contain underlines
                 enum: [one, 2, underscored_param, for]
+
+# NOT WORK YET
```
Comment (lanwen) [inline rarc-example/src/main/resources/api.raml:111]: is this PR WIP now?

Comment (viclovsky) [reply rarc-example/src/main/resources/api.raml:111]: In fact, generation from XSD schema doesn't work in mater. Therefore I'd think that nobody uses it. 
Anyway, I'll open bug ticket and if I have a time, I'll fix it in another Pull Request. Will it work for you?

Comment (lanwen) [reply rarc-example/src/main/resources/api.raml:111]: ok, look forward to new changes

**Review by lanwen — APPROVED** [review]

File: rarc-core/src/main/java/ru/lanwen/raml/rarc/util/JsonCodegen.java [hunk 3]
```diff
@@ -2,18 +2,16 @@
...
 import com.sun.codemodel.JCodeModel;
 import io.restassured.path.json.JsonPath;
-import org.jsonschema2pojo.DefaultGenerationConfig;
-import org.jsonschema2pojo.GenerationConfig;
-import org.jsonschema2pojo.GsonAnnotator;
-import org.jsonschema2pojo.SchemaGenerator;
-import org.jsonschema2pojo.SchemaMapper;
-import org.jsonschema2pojo.SchemaStore;
-import org.jsonschema2pojo.SourceType;
+import org.jsonschema2pojo.*;
```
Comment (lanwen) [inline rarc-core/src/main/java/ru/lanwen/raml/rarc/util/JsonCodegen.java]: ah

**Review by lanwen — CHANGES_REQUESTED** [review]

**Pull request merged**